draft-ietf-quic-transport.md

```diff
@@ -2300,11 +2302,23 @@ linked by any other entity.
 At any time, endpoints MAY change the Destination Connection ID they send to a
 value that has not been used on another path.
 
-An endpoint MUST use a new connection ID if it initiates connection migration as
-described in {{initiating-migration}} or probes a new network path as described
-in {{probing}}.  An endpoint MUST use a new connection ID in response to a
-change in the address of a peer if the packet with the new peer address uses an
-active connection ID that has not been previously used by the peer.
+An endpoint MUST NOT reuse a connection ID when sending from more than one local
+address, for example when initiating connection migration as described in
+{{initiating-migration}} or when probing a new network path as described in
+{{probing}}.
+
+Similarly, an endpoint MUST NOT reuse a connection ID when sending to more than
+one destination address.  Due to network changes outside the control of its
+peer, an endpoint might receive packets from a new source address with the same
+destination connection ID, in which case it MAY continue to use the current
+connection ID with the new remote address while still sending from the same
+local address.
```

Comment:
```suggestion
Due to network changes outside the control of its
peer, an endpoint might receive packets from a new source address with the same
destination connection ID, in which case it MAY continue to use the current
connection ID with the new remote address while still sending from the same
local address. In all other cases, an endpoint MUST NOT reuse a connection ID when sending to more than
one destination address. 
```

Reply:
I don't dislike the flow once flipped, but the first sentences of this and the preceding paragraph are trying to be very clearly the main requirement, with the subsequent items explaining motivation and any exceptions.

This is in response to the original issue we're trying to address, which is that all of these requirements were in place but got lost in the rest of the text, so the idea was to highlight the main requirements in a single place.

Thoughts?

Reply:
I understand the spirit of the change, which is to make the primary requirement properly conditional, but it does so by burying that requirement.  I'm comfortable with the original.  Happy to iterate again later.